postgres-persistence/src/main/java/com/netflix/conductor/postgres/dao/PostgresExecutionDAO.java

```diff
@@ -155,7 +156,7 @@ public List<TaskModel> createTasks(List<TaskModel> tasks) {
                                             + task.getReferenceTaskName()
                                             + ", key="
                                             + taskKey);
-                            return;
+                            continue;
```

Comment:
Should not return early when a task is already scheduled, one of the next ones could still not be.